monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/SkippedEventsTimeline.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+import {Timeline, TimelineEvent} from 'react-event-timeline';
+import { faArrowsAltV } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as PropTypes from 'prop-types';
+
+
+export default class SkippedEventsTimeline extends Component {
+
+  render() {
+    return (
+      <div>
+        <Timeline style={{fontSize: '100%'}}>
+          <TimelineEvent
+            bubbleStyle={{border: '2px solid #ffcc00'}}
```

Comment:
This is just the worst. @ShayNehmad do you have any clue how to import style from the stylesheet and turn it into inline? I struggled with it for quite a long time and couldn't figure it out ;(

Reply:
Give an `id` to the surrounding `div` and then use the `id` in the stylesheet to refer to this?

Reply:
That references the entire timeline event. This component also adds multiple children without any classes, so I'd like to avoid writing css selectors that select n-th child div of n-th child div...

Reply:
Keep it like that, it's a simple enough component